<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anomalous Event Detection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Sequence</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>59</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Video</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Col</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>37</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Row</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_row">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>11</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_col">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>37</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>107</x>
         <y>11</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ini</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>107</x>
         <y>37</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Fin</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_ini">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>11</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_fin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>37</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_record">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Configuration File</string>
      </property>
      <widget class="QTextEdit" name="edt_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_find">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>31</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_load">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>71</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+L</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>119</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Events</string>
      </property>
      <widget class="QPushButton" name="btn_play">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>.</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/play_icon.png</normaloff>img/play_icon.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+P</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>19</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Frame</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_frame">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_playstep">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>20</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/play_icon.png</normaloff>img/play_icon.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_record">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>img/record_icon.png</normaloff>img/record_icon.png</iconset>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Output</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Directory</string>
      </property>
      <widget class="QPushButton" name="btn_find_dir">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>12</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_load_dir">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>12</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Dir</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="edt_dir">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>14</y>
         <width>260</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>49</y>
        <width>351</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Files</string>
      </property>
      <widget class="QTextEdit" name="edt_list_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_generate">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>135</y>
         <width>134</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_token">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>29</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>*.avi</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Finder token</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_rze">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>69</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edt_step">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>109</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Resize</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>211</x>
         <y>90</y>
         <width>22</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
